### ISSUE TITLE
Feature request: undo send

### ISSUE THREAD

**Issue by rmerom** [ethereum/mist#1787]
Wallets in a way are much like an email client. It's a way for people to communicate, only with money in this case.
People make mistakes. And these mistakes can be very costly. While an email gone wrong can have bad consequences, sending a huge amount of ETH because of a momentary distraction or a "fat finger" can have a devastating effect for an individual.
Gmail has introduced [undo send](https://support.google.com/mail/answer/2819488?co=GENIE.Platform%3DDesktop&hl=en) a few years ago. Would Mist consider adding something similar?

**Comment by luclu** [comment]
Good call. I know the mail client [N1](https://www.nylas.com/) to have the undo feature. Basically it just wait a configurable amount of time (10s to 1m) before really sending the mail.

Another way is a staging process. I think that parity is using or tried out such a staging area for transactions to be confirmed by the user after the initial execution. 

I will leave this open as an feature request.
